### PR TITLE
Compile for the browser.

### PR DESCRIPTION
**Pull request by bayleedev**
I used webpack, because I hate myself.

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  entry: './index.js',
+  output: {
+    path: __dirname,
+    filename: 'fuzzyfind.min.js'
+  },
+  plugins: [
+    new webpack.optimize.UglifyJsPlugin({
+      compress: { warnings: false },
+    }),
+    new webpack.BannerPlugin([
```
Comment (amjith) [inline webpack.config.js:13]: What does this do?

Comment (amjith) [inline webpack.config.js:13]: Never mind. I found out what it does. But is this common? Do people really leave big comments on top of minified JS?

Comment (bayleedev) [inline webpack.config.js:13]: MIT says:

```
...
The above copyright notice and this permission notice shall be included in
all copies or substantial portions of the Software.
...
```

Comment (amjith) [inline webpack.config.js:13]: Interesting! Ok, I guess that's fine then.

**Comment by amjith** [comment]
👍

**Pull request merged**